lib/rex/post/meterpreter/extensions/stdapi/sys/process.rb

```diff
@@ -395,7 +395,10 @@ def to_table(opts={})
     # ppid.  Cut columns from the list if they aren't there.  It is conceivable
     # that processes might have different columns, but for now assume that the
     # first one is representative.
-    cols.delete_if { |c| !( first.has_key?(c.downcase) ) or first[c.downcase].nil? }
+    cols.delete_if do |c|
+      !(any? {|r| r.has_key?(c.downcase)}) or
```

Comment:
@egypt: thanks for the review boss, if i knew you were lookin i'd have added a ternary conditional assignment somewhere ;-).

Comments nixed and style changes implemented. I wasn't thrilled with the !(any?..) piece either, the entire block looks voodoo-ish to me, but its hidden deep enough in the guts that anyone digging in there deserves some head scratching.